book/src/rust-client/mint_consume_send_tutorial.md

```diff
@@ -0,0 +1,450 @@
+# Mint, Consume, and Send Assets
+
+*Using the Miden Client in Rust to Mint, Consume, and Send Assets*
+
+## Overview
+
+In the previous section we initialized our repository and covered how to create an account and deploy a faucet. In this section we will mint tokens from the faucet for *Alice*, consume these created notes, and then show how to send assets to other accounts.
+
+## What We'll Cover
+* Minting tokens from a faucet
+* Consuming notes
+* Sending tokens to other users
+
+## Step 1: Minting Tokens from the Faucet
+To mint notes with tokens from the faucet we created, Alice needs to call the faucet with a mint transaction request.
+
+```rust
+let amount: i32 = 100;
+let fungible_asset = FungibleAsset::new(faucet_account.id(), amount)
+    .unwrap();
+
+let transaction_request = TransactionRequest::mint_fungible_asset(
```

Comment:
`TransactionRequest`s and the transaction itself are important concepts here. Can you say a few words above or below the code snippet what the user did here?